examples/css/timetree_chrome_devtools.css

```diff
@@ -1,60 +0,0 @@
-.chrome-devtools-timetree, .chrome-devtools-timetree .label text {
```

Comment:
IMO, we should make this theme the default. Most people just want "good" defaults. They can easily override still. The black/white default currently IMO is too dull and makes people immediately want to change it. I was going to do this, but didnt want to spend the time to figure out what it was overriding, etc. since the chrome theme doesn't work without the default basic one first.